frontend/dist/assets/branding.sample/welcome-simple-text.html

```diff
@@ -5,7 +5,16 @@ <h2 style="margin-bottom: 1em; color: #555">Collaborative information management
   management that has been created over 20 years of experience in the service of collective intelligence.
 </p>
 <p>
-  Tracim is available in: Deutsch, English, Español, Français, Português and العربية
+  Tracim is available in:
+  <ul>
+    <li>Deutsch</li>
+    <li>English</li>
+    <li>Español</li>
+    <li>Français</li>
+    <li>Norsk</li>
+    <li>Português</li>
+    <li>العربية</li>
```

Comment:
it is alphabetical order ? if yes, why arabic is at the end of the list ?

Reply:
> email translation is it also done in weblate ? (I don't remember)

yes

> it is alphabetical order ? if yes, why arabic is at the end of the list ?

yes, to know where the Arabic letters should be in the alphabetical order, we relied on other sites that have the language and on all of them Arabic is at the end of the list.

Reply:
strange but ok.